components/browse/ContentCardSection.vue

```diff
@@ -57,6 +73,17 @@
     computed: {
       cards() {
         return this.section.fields.hasPart.filter(card => card.fields);
+      },
+
+      isPeopleSection() {
```

Comment:
this can be made a bit more concise:
```
      isPeopleSection() {
        if (this.cards.length !== 4) return false;
        return this.cards.every((card) => {
          return card.fields.identifier ? entityParamsFromUri(card.fields.identifier).type === 'person' : false;
        });
      }
```

Reply:
I have applied your changes, but decided to keep the identifier variable in, otherwise i would have repeated `card.fields.identifier` twice.